enterprise/cmd/frontend/auth/githuboauth/session.go

```diff
@@ -83,6 +86,7 @@ func (s *sessionIssuerHelper) GetOrCreateUser(ctx context.Context, token *oauth2
 			CreateIfNotExist:    s.allowSignup,
 		})
 		if err == nil {
+			hubspotutil.SyncUser(verifiedEmail, hubspotutil.SignupEventID, &hubspot.ContactProperties{})
```

Comment:
```suggestion
			go hubspotutil.SyncUser(verifiedEmail, hubspotutil.SignupEventID, &hubspot.ContactProperties{})
```

Not sure if this is necessary, but the only other place started a new goroutine.